<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormReciboBase</class>
 <widget class="QWidget" name="FormReciboBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Recibo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="LNumero">
         <property name="text">
          <string>Numero de Recibo:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Fecha:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="DEFecha">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cliente:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBCliente">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="TETexto"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QGroupBox" name="GBFormaPago">
       <property name="title">
        <string>Forma de Pago</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="RBLuego">
          <property name="text">
           <string>A Pagar Luego</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBContado">
          <property name="text">
           <string>Contado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBotro">
          <property name="text">
           <string>Otro</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>298</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="GBResumen">
       <property name="title">
        <string>Resumen</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Pagado:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDSBPrecio" name="dSBPagado">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="prefix">
             <string>$ </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Deuda Actual:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDSBPrecio" name="dSBDeuda">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="prefix">
             <string>$ </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Total:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="EDSBPrecio" name="dSBTotal">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="prefix">
             <string>$ </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDSBPrecio</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">edsbprecio.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>DEFecha</tabstop>
  <tabstop>CBCliente</tabstop>
  <tabstop>TETexto</tabstop>
  <tabstop>RBContado</tabstop>
  <tabstop>RBotro</tabstop>
  <tabstop>dSBPagado</tabstop>
  <tabstop>dSBDeuda</tabstop>
  <tabstop>dSBTotal</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
